From 6c4e1e18b84f8129fd095783507b5593a9bb8d06 Mon Sep 17 00:00:00 2001
From: chrmhoffmann <chrmhoffmann@gmail.com>
Date: Sat, 4 Aug 2018 20:38:30 +0200
Subject: [PATCH 23/23] lineage: Allow background broadcast for CAMERA_BUTTON

This enables launch of camera app (snap) when pressing the camera button.

Change-Id: I76fd622031f628b20212e8c7a6239b49fd121a2e
---
 config/permissions/lineage-sysconfig.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config/permissions/lineage-sysconfig.xml b/config/permissions/lineage-sysconfig.xml
index 4ab0f696..3330b1b8 100644
--- a/config/permissions/lineage-sysconfig.xml
+++ b/config/permissions/lineage-sysconfig.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2017 The LineageOS Project
+<!-- Copyright (C) 2017-2018 The LineageOS Project
 
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
@@ -18,6 +18,7 @@
 
     <!-- Broadcast actions that are currently exempted from O+ background
          delivery restrictions. -->
+    <allow-implicit-broadcast action="android.intent.action.CAMERA_BUTTON" />
     <allow-implicit-broadcast action="lineageos.intent.action.INITIALIZE_LINEAGE_HARDWARE" />
     <allow-implicit-broadcast action="lineageos.intent.action.LID_STATE_CHANGED" />
 
-- 
2.17.1

